sdk/communication/communication-identity/review/communication-identity.api.md

```diff
@@ -21,28 +21,47 @@ export class CommunicationIdentityClient {
     constructor(connectionString: string, options?: CommunicationIdentityClientOptions);
     constructor(endpoint: string, credential: KeyCredential, options?: CommunicationIdentityClientOptions);
     constructor(endpoint: string, credential: TokenCredential, options?: CommunicationIdentityClientOptions);
-    createUser(options?: OperationOptions): Promise<CommunicationUserIdentifier>;
+    createUser(options?: CreateUserOptions): Promise<CommunicationUserIdentifier>;
```

Comment:
Instead of changing this function we can introduce a new `getOrCreateUser` function. That then could return the same response type as the `getUser` function? That means the service has to include lastTokenIssuedAt on the POST route as well for existing users.

Reply:
We then need to decide if `externalId` should be mandatory for this function.

Reply:
Hm, and then we would also need a `getOrCreateUserAndToken` function or bundle it all in one. Maybe keeping it as you have it is less confusing after all.

Reply:
I followed the getOrCreateUser pattern here: https://github.com/Azure/azure-sdk-for-net/pull/48447/files

Reply:
Adjusted https://github.com/Azure/azure-sdk-for-net/pull/48447/files to use createUser and createUserAndToken